tests/test_load_video_frames.py

```diff
@@ -448,7 +448,7 @@ def test_megadetector_lite_yolox_dog(tmp_path):
             "-vcodec",
             "libx264",
             "-crf",
-            "25",
+            "23",
```

Comment:
set CRF to default (otherwise test fails due to lossy compression in creating the test video): https://trac.ffmpeg.org/wiki/Encode/H.264